src/ItemsList.js

```diff
@@ -0,0 +1,23 @@
+import Item from './Item';
+
+function ItemsList({ items, removeItem, changeItemStatus }) {
+    return (
+        <div className='content'>
+            <ul className='list' id='list'>
+                {items.map(item => {
+                    return (
+                        <Item
+                            className={`list__item` + (item.isChecked ? ' list__item--checked' : ' rhyryh')}
```

Comment:
what `rhyryh` means?

Reply:
I used it for testing. I forgot to delete it.